dora/core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -98,6 +102,67 @@ public static void changeLocalFilePermission(String filePath, String perms) {
     }
   }
 
+  /**
+   * Changes local file's user defined file attribute.
+   * @param filePath the file to change attribute
+   * @param name the name of attribute
+   * @param value the value of attribute
+   */
+  public static void changeLocalFileUserDefinedAttribute(String filePath, String name,
+                                                         byte[] value) {
+    try {
+      UserDefinedFileAttributeView attributeView =
+          Files.getFileAttributeView(Paths.get(filePath), UserDefinedFileAttributeView.class);
```

Comment:
can you add a comment to explain what is an attribute view what is this for?

Reply:
is the attr only set locally and will lose after a worker restarts?

Reply:
> is the attr only set locally and will lose after a worker restarts?
No, it sets the attribute to the local file system, so it will not change after a worker restarts. (works in CentOS, not work in MacOS)